lib/ansible/module_utils/lxd.py

```diff
@@ -110,7 +109,10 @@ def _send_request(self, method, url, body_json=None, ok_error_codes=None, timeou
             body = json.dumps(body_json)
             self.connection.request(method, url, body=body)
             resp = self.connection.getresponse()
-            resp_json = json.loads(resp.read())
+            resp_data = resp.read()
+            if not PY2:
+                resp_data = resp_data.decode('utf-8')
+            resp_json = json.loads(resp_data)
```

Comment:
Note: I believe that both python2 and python3's json.loads() can handle resp_data as unicode strings.  So you should be able to do this like this:
``` python
from ansible.module_utils._text import to_text

[...]
resp_data = resp.read()
resp_data = to_text(resp_data, errors='surrogate_or_strict')
resp_json = json.loads(resp_data)
```

to_text is a nice helper because it allows much better control over tracebacks and won't touch the value if resp_data is already a text string (whereas resp_data.decode could traceback if resp_data is already a text string for some strange reason)